src/main/java/seedu/taskify/logic/commands/CommandResult.java

```diff
@@ -2,23 +2,20 @@
 
 import static java.util.Objects.requireNonNull;
 
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Objects;
 
 /**
  * Represents the result of a command execution.
  */
 public class CommandResult {
 
-    private static boolean showHome = true;
-    private static boolean showExpired = false;
-    private static boolean showCompleted = false;
-    private static boolean showTodays = true;
-    private static boolean showUncompleted = false;
-    private static final String ALREADY_IN_HOMETAB = "You are already in home tab!";
-    private static final String ALREADY_IN_EXPIREDTAB = "You are already in expired tab!";
-    private static final String ALREADY_IN_COMPLETEDTAB = "You are already in completed tab!";
-    private static final String ALREADY_IN_UNCOMPLETEDTAB = "You are already in uncompleted tab!";
-
+    private static ArrayList<Boolean> showTabBoolean = new ArrayList<>(Arrays.asList(true, false, false, false));
+    private static final int HOME = 0;
+    private static final int EXPIRED = 1;
+    private static final int COMPLETED = 2;
+    private static final int UNCOMPLETED = 3;
```

Comment:
Looks ok, but I'm not sure if they will prefer using an enum. Idk

Reply:
I think this is simpler to understand.